### ISSUE TITLE
password

### ISSUE THREAD

**Issue by rmendez-uala** [rstudio/distill#346]
Hello everyone! is it possible to put a password to an article?

**Comment by jmbuhr** [comment]
Not quite. But there are currently 2 R packages that attempt something like password protected Rmarkdown html documents. The first is less secure but might also work with distill articles, the second is more secure but I believe only works with standard Rmarkdown html documents:

https://github.com/favstats/rmdprotectr
https://github.com/dirkschumacher/encryptedRmd/

**Comment by jmbuhr** [comment]
Update: The first doesn't work with distill because it is to easy for distill to find the file (because it doesn't really password encrypt the file, it just hides it in a subfolder, which might  be sufficient for certain applications and webservers but not here)

**Comment by jmbuhr** [comment]
On closer inspection, the second option is not straightforward for distill either, In this case because it is encrypted too well. While you can password-protect any html-file with `encryptedRmd::encrypt_html_file`, distill then can't list that article in the listings page (because it is no longer an article), but you can add a link to an encrypted article inside of a distill article.

**Comment by jmbuhr** [comment]
e.g. In the folder for the article put another Rmd document `secret.Rmd` with the yml option: `output: encryptedRmd::encrypted_html_document`. Then, in the regular article, put a link to the encrypted version:
`[Top Secret](./secret.html.enc.html)`
And make sure to remove the unencrypted `secret.html` and the key-file before rebuilding the side (but store the key-file somewhere safe)
